src/main/java/nl/esciencecenter/restape/IOUtils.java

```diff
@@ -29,7 +29,7 @@ public class IOUtils {
         /**
          * URL to the README file containing instructions on how to run the workflows.
          */
-        private static final String README_URL = "https://raw.githubusercontent.com/Workflomics/containers/add_instructions/instructions.txt";
+        private static final String README_URL = "https://raw.githubusercontent.com/Workflomics/tools-and-domains/refs/heads/add_instructions/instructions.txt";
```

Comment:
Maybe not for now, but isn't this link going to break again when the branch it's refering to is deleted?

Reply:
This is pointing to a branch that should not be merged into main. This is not the best way to deal with it, but I just didn't want the text to be hardcoded. This should be refactored.

Reply:
I made an issue out of it https://github.com/sanctuuary/restape/issues/74